src/relay/pass/type_infer.cc

```diff
@@ -61,6 +61,19 @@ TVM_REGISTER_API("tvm.relay.type_relation.TupleGetItem")
 .set_body_typed<bool(const Array<Type>&, int, const Attrs&, const TypeReporter&)>(
     TupleGetItemRel);
 
+struct ResolvedTypeInfo {
+  explicit ResolvedTypeInfo(Type checked_type, Array<Type> type_args)
+      : checked_type(checked_type), type_args(type_args) {}
+  ResolvedTypeInfo(const ResolvedTypeInfo& rti)
```

Comment:
likely you don't need this, as long as there are default constructor with no argument